mime-types-redirector/lib/mime-types.rb

```diff
@@ -0,0 +1,13 @@
+require 'mini_mime'
+
+module MIME
+  class Types
+    def self.[] type
+      Array.wrap MiniMime.lookup_by_content_type(type)
+    end
+
+    def self.type_for filename
+      Array.wrap MiniMime.lookup_by_filename(filename)
```

Comment:
@Fryguy Well, I was going off of this change:

https://github.com/rest-client/rest-client/pull/557/files#diff-3724e1adc91ded3d8e979d73f91f1b1eL915

So I don't think so, but maybe I read that diff wrong (it was Monday morning at the time of writing this code...)